files/es/glossary/base64/index.md

```diff
@@ -250,34 +290,33 @@ function strToUTF8Arr (sDOMStr) {
     } else if (nChr < 0x10000) {
       /* tres bytes */
       aBytes[nIdx++] = 224 + (nChr >>> 12);
-      aBytes[nIdx++] = 128 + (nChr >>> 6 & 63);
+      aBytes[nIdx++] = 128 + ((nChr >>> 6) & 63);
       aBytes[nIdx++] = 128 + (nChr & 63);
     } else if (nChr < 0x200000) {
       /* cuatro bytes */
       aBytes[nIdx++] = 240 + (nChr >>> 18);
-      aBytes[nIdx++] = 128 + (nChr >>> 12 & 63);
-      aBytes[nIdx++] = 128 + (nChr >>> 6 & 63);
+      aBytes[nIdx++] = 128 + ((nChr >>> 12) & 63);
+      aBytes[nIdx++] = 128 + ((nChr >>> 6) & 63);
       aBytes[nIdx++] = 128 + (nChr & 63);
     } else if (nChr < 0x4000000) {
       /* cinco bytes */
       aBytes[nIdx++] = 248 + (nChr >>> 24);
-      aBytes[nIdx++] = 128 + (nChr >>> 18 & 63);
-      aBytes[nIdx++] = 128 + (nChr >>> 12 & 63);
-      aBytes[nIdx++] = 128 + (nChr >>> 6 & 63);
+      aBytes[nIdx++] = 128 + ((nChr >>> 18) & 63);
+      aBytes[nIdx++] = 128 + ((nChr >>> 12) & 63);
+      aBytes[nIdx++] = 128 + ((nChr >>> 6) & 63);
       aBytes[nIdx++] = 128 + (nChr & 63);
-    } else /* if (nChr <= 0x7fffffff) */ {
+    } /* if (nChr <= 0x7fffffff) */ else {
```

Comment:
Not sure about this, I think the intention is read it like an  `else if`

```suggestion
    } else /* if (nChr <= 0x7fffffff) */ {
```

Reply:
This file also looks like it's quite out-of-sync with upstream, but I've made a tweak so we can get this PR landed!